// Copyright (c) 2020 Boston Dynamics, Inc.  All rights reserved.
//
// Downloading, reproducing, distributing or otherwise using the SDK Software
// is subject to the terms and conditions of the Boston Dynamics Software
// Development Kit License (20191101-BDSDK-SL).

syntax = "proto3";

package bosdyn.api;

option java_outer_classname = "SynchronizedCommandProto";

import "bosdyn/api/mobility_command.proto";

message SynchronizedCommand {
    message Request {
        MobilityCommand.Request mobility_command = 2;
    }

    message Feedback {
        MobilityCommand.Feedback mobility_command_feedback = 2;
    }
}